frontend/src/views/home.rs

```diff
@@ -32,21 +32,88 @@ pub fn home() -> Html {
     let password_is_valid = use_state(|| true);
 
     log::info!("registration_form {:?}", &registration_form.clone());
-    // let onsubmit = { !todo(); };
+    let onsubmit = {
+        let registration_form = registration_form.clone();
+
+        let username = username.clone();
+        let first_name_ref = first_name_ref.clone();
+        let last_name_ref = last_name_ref.clone();
+        let email_ref = email_ref.clone();
+        let password_ref = password_ref.clone();
+        let confirm_password_ref = confirm_password_ref.clone();
+
+        let password_is_valid = password_is_valid.clone();
+
+        Callback::from(move |event: SubmitEvent| {
+            event.prevent_default();
+            log::info!("registration_form {:?}", &registration_form.clone());
+
+            let username = username.cast::<HtmlInputElement>().unwrap().value();
+            let first_name = first_name_ref.cast::<HtmlInputElement>().unwrap().value();
+            let last_name = last_name_ref.cast::<HtmlInputElement>().unwrap().value();
+            let email = email_ref.cast::<HtmlInputElement>().unwrap().value();
+            let password = password_ref.cast::<HtmlInputElement>().unwrap().value();
+            let confirm_password = confirm_password_ref
+                .cast::<HtmlInputElement>()
+                .unwrap()
+                .value();
+
+
+            // Fix: 
+            // Check password against DB entry -----            
```

Comment:
what u mean here

Reply:
It was more of a note-for-later that didn't need to be there